tests/records/test_relations_languages.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test record relationships."""
+
+import pytest
+from invenio_access.permissions import system_identity
+from invenio_db import db
+from invenio_records.systemfields.relations import InvalidRelationValue
+from invenio_vocabularies.proxies import current_service as vocabulary_service
+from invenio_vocabularies.records.api import Vocabulary
+from jsonschema.exceptions import ValidationError
+
+from invenio_rdm_records.records.api import RDMDraft, RDMRecord
+
+
+#
+# Tests
+#
+def test_languages_field(app, location, minimal_record):
+    """Languages should be defined as a relation."""
+    assert 'languages' in RDMDraft.relations
+    assert 'languages' in RDMRecord.relations
+    assert RDMDraft.relations.languages
+
+
+def test_languages_validation(app, location, minimal_record, lang):
+    """Test languages relationship."""
+    minimal_record["metadata"]["languages"] = [{"id": "eng"}]
+    draft = RDMDraft.create(minimal_record)
+    draft.commit()
+    db.session.commit()
+    assert minimal_record["metadata"]["languages"] == [{"id": "eng"}]
+
+    lang = list(draft.relations.languages())[0]
```

Comment:
Hmm I thought the descriptors and so on were to make `draft.relations.languages[0]` possible...  And this index is a `Vocabulary` (which should be read as a Vocabulary entry) right?

Reply:
I think is an iterator, thats why the cast to `list`... otherwise needs the `next()`?